MetaMask/metamask-mobile
# Android deeplink approve modal changes network and works

**Description**
When deeplink to send funds it's pressed and it's needed to change the network, it would only change the network and do not show the approve modal.

**Proposed Solution**
The approve modal appears
Remove unused code


**Code impact**
Low

**Screenshots/Recordings**
Recordings of solution:
With two devices (Huawei P30 pro and Samsung A22, both Android 12)
https://recordit.co/NpxyGiXbVD
With one device:
https://user-images.githubusercontent.com/46944231/179568171-cb51ae76-c784-43bb-bedf-8b584cbdd06f.mp4

**Test Cases**
Case1:
* Select ethereum mainnet as your network
* Open this [deeplink](https://metamask.app.link/send/0x3dD3DfaAdA4d6765Ae19b8964E2BAC0139eeCb40@4/approve?address=0x178e3e6c9f547A00E33150F7104427ea02cfc747&uint256=5e8)
* Notice that the network changed and the modal is openning 

**Issue**

Progresses #3880 

**Checklist**
